Framework/PythonInterface/plugins/algorithms/MatchPeaks.py

```diff
@@ -19,18 +19,18 @@ def mask_ws(ws_to_mask, xstart, xend):
 
     if xstart > 0:
         logger.debug('Mask bins smaller than {0}'.format(xstart))
-        MaskBins(InputWorkspace=ws_to_mask, OutputWorkspace=ws_to_mask, XMin=x_values[0], XMax=x_values[xstart])
+        MaskBins(InputWorkspace=ws_to_mask, OutputWorkspace=ws_to_mask, XMin=x_values[0], XMax=x_values[int(xstart)])
```

Comment:
Unfortunately that won't work because we later assume it's a double when writing it to a TableWorkspace, see https://github.com/mantidproject/mantid/blob/master/Framework/PythonInterface/plugins/algorithms/MatchPeaks.py#L210